src/components/forms/riddleUpsertForm/CreateRiddleForm.tsx

```diff
@@ -45,10 +47,42 @@ export const CreateRiddleForm = () => {
 
 	const user = useLoggedInUser();
 
+	const uploadAllImages = useCallback(async (data: RiddleUpsertDetail) => {
+		// Riddle image
+		if (data.imageFile) {
+			const imageRef = ref(storage, `images/${data.imageFile.name + uuidv4()}`);
+			const url = await uploadBytes(imageRef, data.imageFile)
+				.then(snapshot => snapshot.metadata.fullPath)
+				.then(() => getDownloadURL(imageRef));
+			data.image = url;
+			delete data.imageFile;
+		}
+
+		// Solved image
+		if (data.solvedImageFile) {
+			const solvedImageRef = ref(
+				storage,
+				`images/${data.solvedImageFile.name + uuidv4()}`
+			);
+			const url = await uploadBytes(solvedImageRef, data.solvedImageFile)
+				.then(snapshot => snapshot.metadata.fullPath)
+				.then(() => getDownloadURL(solvedImageRef));
+			data.solvedImage = url;
+			delete data.solvedImageFile;
+		}
+
+		// Questions images
+		// TODO?
```

Comment:
Zatím lze nahrát pouze `image` a `solvedImage`. Tady bych to klidně prozatím nechal a s ohledem na to, jak budeme stíhat, bych to pak dodělal i pro otázky. Když nebudeme stíhat, tak prostě budou otázky bez obrázku. :D

Zase je to jen naše vůle a když to tam nebude, tak nám za to nikdo hlavu neutrhne. Ale jak říkám, když budeme stíhat, nemám s tím problém! :)